schemas/openapi.json

```diff
@@ -3866,6 +3866,18 @@
             ],
             "title": "Metadata",
             "description": "Metadata for the annotation"
+          },
+          "identifier": {
+            "anyOf": [
+              {
+                "type": "string"
+              },
+              {
+                "type": "null"
+              }
+            ],
+            "title": "Identifier",
+            "description": "The identifier of the annotation"
```

Comment:
maybe add a description that says if sent in an annotation is updated, if unset one is created for you